tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/Deduplication/DeduplicationTests.cs

```diff
@@ -23,6 +23,8 @@ public DeduplicationTests(ITestOutputHelper output)
         : base("Deduplication", output)
     {
         SetServiceVersion("1.0.0");
+
+        SetEnvironmentVariable("DD_TRACE_DEBUG", "1");
```

Comment:
oops 😄 
```suggestion
```

Reply:
Yeah, you are right. That was one of the tests failing by the sneaky CS crash...